python/nano/test/pytorch/tests/inference/test_bf16_ipex.py

```diff
@@ -354,18 +354,20 @@ def test_ipex_jit_channels_last_3d_inference(self):
         model = DummyModelWith3d()
         x1 = torch.rand(32, 3, 3, 224, 224) # 5-dim input test
         x2 = 3
-        # TODO: quantize will fail here, will fix it in next PR
-        ipex_jit_channels_last_model = InferenceOptimizer.trace(model, accelerator="jit", 
-                                                                use_ipex=True, precision='bf16',
-                                                                input_sample=(x1, x2),
-                                                                enable_onednn=True,
-                                                                channels_last=True)
+        ipex_jit_channels_last_model = InferenceOptimizer.quantize(model,
+                                                                   accelerator="jit", 
+                                                                   use_ipex=True,
+                                                                   precision='bf16',
```

Comment:
just a thought, should we add some error message to users who use `precision` in `trace` in future PR?

Reply:
Good suggestion! I will add such behavior in next PR.